README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
+
 **b)** `"1" === 1`
 
+False - 3 ==='s are looking for a 100% match, and string 1 is not exactly the same as number 1.
+Console says: False
+
 **c)** `x == 'x'`
 
+True - Again, because double ='s are looking to see a close fit, rather than an exact match, and x is similar to string x. Moreover, javascript will turn x into a string in an effort to 'help out'
+Console says: Uncaught (x is not defined)
+After defining x as 1, console says: false, my assumption being that variable x is not == string x.
+
 **d)** `x == (x+'')`
```

Comment:
1d) There are a couple of edge cases here: if `var x = NaN` it will be false, as `NaN` is not equal to anything, including itself, and if x is a boolean, it will be autoconverted to a string when added to a string.